src/Components/Search/NewSearch/NewSearchInputPills.tsx

```diff
@@ -157,6 +166,10 @@ export const NewSearchInputPillsContainer = () => {
     }
   }
 
+  const onPillClick = (pill: PillType) => {
+    setSelectedPill(pill)
+  }
+
   return (
     <Flex alignItems="center" bg="white">
       <Flex position="absolute" left={0} opacity={showPreviousChevron ? 1 : 0}>
```

Comment:
@rajsam003 we need to change the `pointer-events` or remove/unmount component (as suggested [here](https://github.com/artsy/force/pull/12276#discussion_r1172844949)), otherwise it may be impossible to click on the "Top" pill due to the fact that the left chevron button is displayed and it will handle the click event (since we only change the `opacity` prop of the component. But we can still click on this chevron button)

https://user-images.githubusercontent.com/3513494/234380859-54cefebe-a100-4a3a-be93-17f448ce4f14.mp4

Reply:
Would [`visibility: hidden`](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility) help here? I thought maybe so, but it seems Flex doesn't support that 🤔

Reply:
I fixed this problem using a z-index property, @dimatretyak let's pair on refactoring in! I tried to follow `{showNextChevron && ()}` approach, but stumbled upon some issues